### ISSUE TITLE
Feature request - Date calculations

### ISSUE THREAD

**Issue by derrickwheals** [gtg922r/obsidian-numerals#55]
This is an awesome plugin which I use almost daily. What would make it even more awesome would be to be able to do date calculations (e.g. add  x days to a date).

Thanks for making such a useful plugin!

**Comment by freqyfreqy** [comment]
Agreed! I love this plugin and date math would be awesome. Numi and Soulver have slightly more features but integration with Obsidian is awesome. Love this plugin Thanks @gtg922r!!!

**Comment by drewtu2** [comment]
+1

**Comment by businessBoris** [comment]
Add my name to the list, please. I don't mind if it's restrictive (ISO only, say).
